devito/ops/operator.py

```diff
@@ -95,7 +97,7 @@ def _specialize_iet(self, iet, **kwargs):
             have the same number of dimensions"
 
         self._headers.append(namespace['ops_define_dimension'](dims[0]))
-        self._includes.append('stdio.h')
+        self._includes.extend(['stdio.h', 'ops_seq.h'])
```

Comment:
`ops_seq.h` ? what is it used for?

Reply:
It is responsible for implementing the OPS API calls. It is necessary to have it.

Reply:
ok